servers/quarkus-server/src/test/java/org/projectnessie/server/error/TestNessieError.java

```diff
@@ -50,12 +51,12 @@
     QuarkusTestProfileInmemory.class) // use the QuarkusTestProfileInmemory, as it can be reused
 class TestNessieError {
 
-  static String baseURI = "http://localhost:19121/api/v1/nessieErrorTest";
-
   private static HttpClient client;
 
   @BeforeAll
   static void setup() {
+    String baseURI = QuarkusTestUtils.getNessieApiV1Url() + "/nessieErrorTest";
```

Comment:
Technically the `nessieErrorTest` resource has nothing to do with Nessie API versions, it's a standalone service. Could we have a "get base URL" method of something like that?

Reply:
Well, my comment was just about the name of the method used for getting the URL :)